rest/server_context.go

```diff
@@ -664,7 +664,7 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 			}
 
 			if dbcontext.RevsLimit < 100 {
-				base.Warnf(base.KeyAll, "Setting the revs_limit (%v) to less than 100 may have unwanted results when documents are frequently updated. Please see documentation for details.", dbcontext.RevsLimit)
+				base.Warnf(base.KeyAll, "Setting the revs_limit (%v) to less than 100, whilst having allow conflicts set to true, may have unwanted results when documents are frequently updated. Please see documentation for details.", dbcontext.RevsLimit)
```

Comment:
Would be nice to change this to `allow_conflicts`, to make it clear what needs to be changed.